Basic cleanup

FlameArrow57 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
This PR does some basic cleanup for Flock code.

Changes 0s and 1s to FALSE and TRUE.

Adds defines for Flockdrone resource costs.

Removes a bit of closet code that wasn't necessary any more with upstream merge.

Removes some unnecessary layers set for Flockmind and Flocktrace.

Makes deconstruction a bit more readable and uses else if rather than only ifs.

Removes a bit of code that didn't make sense.

Removes stat panels from Flockmind, Flocktrace, and Flockdrones.

Removes comments that seemed unnecessary.

Other super minor stuff.


## Why's this needed? <!-- Describe why you think this should be added to the game. -->
Improved code style and removal of unnecessary stuff.

TobleroneSwordfish commented on code/datums/abilities/flock/flockmind.dm:
```diff
@@ -266,7 +264,7 @@
 		if(A.canAIControl())
 			targets += A
-	if(targets.len > 1)
+	if(length(targets))
```
those are not equivalent

FlameArrow57 replied:
I see they're not now, but maybe it can be changed or it's a bug? Not sure why the ability wouldn't be used for if there's a single door available.

amylizzle replied:
yeah, I'd say we keep your change. The original looks like an oversight to me.

TobleroneSwordfish reviewed (changes requested):
very cool, thank you Flame!

amylizzle reviewed (approved):
All looks good :+1:

stonepillars commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -24,19 +24,19 @@
+	is_npc = TRUE
 
-	use_stamina = 0 //haha no
+	use_stamina = FALSE //haha, no
```
haha, no
```suggestion
	use_stamina = FALSE
```

stonepillars commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -24,19 +24,19 @@
+	use_stamina = FALSE //haha, no
 
-	can_lie = 0 // no rotate when dead
+	can_lie = FALSE // no rotate when dead
```
```suggestion
	can_lie = FALSE
```
and probably the other redundant/snarky comments

stonepillars commented on code/mob/living/intangible/flock/flockmind.dm:
```diff
@@ -35,25 +34,25 @@
+		<br><span class='bold'>System Integrity:</span> [round(src.flock.total_health_percentage()*100)]%
+		<br><span class='bold'>Cognition:</span> COMPUTATIONAL NEXUS
+		<br>###=-</span></span>"}
+	else
+		return null // give the standard description
```
```suggestion
```
redundant. also probably make the if up there an early return

stonepillars commented on code/obj/flock/furniture.dm:
```diff
@@ -177,18 +177,20 @@
-				else
-					return ..()
-			else if(user.drop_item())
+			if(user.drop_item())
```
Isn't there some fuckery with drop_item still needing u_unequip or something? I remember needing to do it anyway and then pali being really confused as to why

FlameArrow57 replied:
Idk, I just changed this because leah made an upstream pr to fix critters not being able to put items in storage containers, which seemed to make this not necessary

TobleroneSwordfish replied:
since it's now identical to the parent implementation it should be fine

FlameArrow57 commented:
Okay uh I just removed comments that seemed unnecessary/self-explanatory as well as obviously not needed ones, feel free to say if I removed one that you thought should be kept

TobleroneSwordfish commented:
Not sure how I feel about just removing every comment that isn't on a variable definition, while most of them are pretty useless, some of them provide information about intent etc.
![image](https://user-images.githubusercontent.com/20713227/169310683-c976db1f-2d48-4c9c-bf70-10fef4df945c.png)
![image](https://user-images.githubusercontent.com/20713227/169311432-b3b5bb2e-31de-4af1-9a9a-448ca6982807.png)

FlameArrow57 commented:
Sure, just comment on any cases that you feel should be kept. Though, for some cases like flock_convert_turf(), that's the only place where it's commented and it's not commented for any other uses of it.

amylizzle commented:
One more thing I think should be included in this: the 
```
//TEMPORARY TEMPORARY
```
stat panel stuff. I already tested just removing it without issue, and it hasn't been useful since the TGUI panel was finished.

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -549,13 +549,12 @@ butcher
 
 /datum/aiTask/sequence/goalbased/rummage/precondition()
 	var/mob/living/critter/flock/drone/F = holder.owner
-	return !(F?.absorber?.item) //only go rummaging if you ain't got stuff to eat
```
useful comment

FlameArrow57 replied:
Idk, seems explained by what's being returned?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -586,22 +585,19 @@ butcher
 		var/mob/living/critter/flock/drone/F = holder.owner
 		usr = F // don't ask, please, don't
 		if(F?.set_hand(1)) // grip tool
-			if(!F.is_in_hands(container_target)) //if it's not in any of our hands, pick it up
```
useful comment

FlameArrow57 replied:
Idk, any way this could be more descriptive? Didn't really explain why item is dropped and then picked up

amylizzle replied:
We drop anything that isn't the target item, so as not to hit the target item with the thing we're holding. Comment could say that?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -586,22 +585,19 @@ butcher
 		usr = F // don't ask, please, don't
 		if(F?.set_hand(1)) // grip tool
-			if(!F.is_in_hands(container_target)) //if it's not in any of our hands, pick it up
-				F.drop_item() // drop whatever we're holding
```
useful comment

FlameArrow57 replied:
Idk, think drop_item() already explains this?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -586,22 +585,19 @@ butcher
+				F.drop_item()
 				F.set_dir(get_dir(F, container_target))
 				F.hand_attack(container_target) //try and pick it up
-			if(F.is_in_hands(container_target)) //did we do it?
```
useful comment

FlameArrow57 replied:
Idk, seems like is_in_hands() kind of already explains this except for intent, maybe would be better to just clarify that this won't work for items that have HUD's open or something?

amylizzle replied:
Honestly this whole block is a nightmare, so I just wanted to explain at every step what is happening

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -586,22 +585,19 @@ butcher
-			if(F.is_in_hands(container_target)) //did we do it?
+			if(F.is_in_hands(container_target))
 				if(F.absorber.item != container_target) //if it's in our manipulating hand
-					// we've picked up a container, just eat it, dump it onto the floor
```
useful comment

FlameArrow57 replied:
Merged into existing comment if seems fine

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -586,22 +585,19 @@ butcher
 				if(F.absorber.item != container_target) //if it's in our manipulating hand
-					// we've picked up a container, just eat it, dump it onto the floor
 					container_target.MouseDrop(get_turf(F)) //this will do nothing on a locked secure storage
-					// might as well eat the container now
```
useful comment

FlameArrow57 replied:
I kind of merged this into the new comment

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -668,12 +663,10 @@ butcher
 			else
 				F.empty_hand(1) // drop whatever we might be holding just in case
-				// grab the item
-				F.set_dir(get_dir(F, harvest_target)) //look at it
```
useful comment

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -715,7 +706,6 @@ butcher
 	var/list/dummy_params = list("icon-x" = 16, "icon-y" = 16)
 
 /datum/aiTask/timed/targeted/flockdrone_shoot/proc/precondition()
-	//if we know there are enemies, and gun is charged and ready
```
useful comment

FlameArrow57 replied:
Maybe would be better as precondition comment?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -808,7 +798,7 @@ butcher
 
 /datum/aiTask/sequence/goalbased/flockdrone_capture/precondition()
 	var/mob/living/critter/flock/F = holder.owner
-	return F?.can_afford(15) && (length(F.flock?.enemies)) //gotta have enemies if we're gonna cage em
```
useful comment

FlameArrow57 replied:
Maybe would be better as precondition comment?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -851,9 +838,9 @@ butcher
-	if(!F.can_afford(15))
+	if(!F.can_afford(FLOCK_CAGE_COST))
 		return TRUE
-	if(get_dist(F, holder.target) > 1) //moved away before we could finish
```
useful comment

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -866,8 +853,7 @@ butcher
 			if(isliving(T))
 				var/mob/living/M = T
 				if(!is_incapacitated(M))
-					// target is up, abort
```
both useful comments

FlameArrow57 replied:
I tried clarifying intent some more, looks okay?

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -878,7 +864,7 @@ butcher
 				owncritter.set_dir(get_dir(owncritter, holder.target))
 				owncritter.hand_attack(holder.target)
 		else
-			holder.interrupt() //somehow lost target, go do something else
```
useful comment

amylizzle commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -979,9 +965,9 @@ butcher
-	if(!F.can_afford(25))
+	if(!F.can_afford(FLOCK_BARRICADE_COST))
 		return TRUE
-	if(get_dist(F, holder.target) > 1) //moved away before we could finish
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -104,23 +98,22 @@
 		boutput(pilot, "<span class='alert'>This drone is already being controlled.</span>")
 		return
 	src.controller = pilot
-	src.ai.stop_move() //cancel any pathing that's happening
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -104,23 +98,22 @@
-	src.is_npc = 0
-	src.dormant = 0
-	src.anchored = 0
-	// move mind into flockdrone
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -391,12 +370,12 @@
 			if (src.emote_check(voluntary, 50))
 				playsound(src, "sound/misc/flockmind/flockdrone_grump[pick("1","2","3")].ogg", 30, 1)
 				return "<b>[src]</b> beeps grumpily[(param? " at [param]!" : "!")]"
-		if ("fart") // i cannot ignore my heritage any longer
```
CRITICAL comment - very important, must keep.

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -604,23 +582,20 @@
 		// do normal movement
 		return ..(NewLoc, direct)
 
-// catchall for shitlisting a dude that attacks us
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -604,23 +582,20 @@
 	if(isflock(enemy))
 		return
 	if(!isdead(src) && src.is_npc && src.flock)
-		// if this is a new guy to add to our enemies, call it out
```
useful comment

FlameArrow57 replied:
Kind of explained by code already?

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -638,18 +613,15 @@
 		var/list/reagent_list = I.reagents.reagent_list
 		for(var/reagent_id in reagent_list)
 			var/datum/reagent/current_reagent = reagent_list[reagent_id]
-			// if the reagent mix dumped on us includes a combustible or harmful reagent, the mob has harmful intent
```
useful comment

FlameArrow57 replied:
I think currently explained by the type check nearby?

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -638,18 +613,15 @@
 		for(var/reagent_id in reagent_list)
 			var/datum/reagent/current_reagent = reagent_list[reagent_id]
-			// if the reagent mix dumped on us includes a combustible or harmful reagent, the mob has harmful intent
-			// (there's other reagents that might be effective on these things without them realising it's dangerous outright)
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -661,15 +633,13 @@
 		return
 	var/prev_damaged = src.damaged
 	if(!isdead(src) && src.is_npc)
-		// if we've been damaged a new stage, call it out
```
useful comment

FlameArrow57 replied:
I left part of it, think saying was already explained by the say()

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -783,40 +751,41 @@
 	if(src.controller)
 		src.release_control()
 	src.flock?.removeDrone(src)
-	// create the flockbits
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -783,40 +751,41 @@
-	// create the flockbits
-	animate_flock_drone_split(src)
-	var/mob/living/critter/flock/bit/B
-	// get candidate places to move them
```
useful comment

FlameArrow57 replied:
Thinking about this some more, think it should be using is_flock_blocked_turf(), not is_blocked_turf() for bits?

amylizzle replied:
I'm fine with that, there's probably an instance or two where that needs changing

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -783,40 +751,41 @@
 		src.flock?.registerUnit(B)
 		SPAWN(0.2 SECONDS)
 			B.set_loc(pick(candidate_turfs))
-	// drop any resources the drone has
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -829,11 +798,11 @@
 	if(src.organHolder)
 		var/obj/item/organ/brain/B = src.organHolder.get_organ("brain")
 		if(B)
-			. += B // always drop brain
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -829,11 +798,11 @@
 	// handle our contents, such as whatever item we're trying to eat or what we're holding
 	for(var/atom/movable/O in src.contents)
 		if(istype(O, /atom/movable/screen))
-			continue // no UI elements please
```
useful comment

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -922,7 +891,7 @@
 	if (!istype(user))
 		return
 
-	if(istype(target,/obj/critter)) //gods how I hate /obj/critter
```
not useful, but I just really hate /obj/critter and I need people to know this.

amylizzle commented on code/mob/living/critter/flock/flockdrone.dm:
```diff
@@ -1145,7 +1111,7 @@
 /datum/equipmentHolder/flockAbsorption/on_unequip()
 	var/obj/item/temp = item
 	if(temp)
-		animate(temp) // cancel animation
```
useful comment

FlameArrow57 replied:
I removed it as I think animate(...) is used in a lot of places with no comment

amylizzle replied:
It is, but it ought not to be imo

amylizzle commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -19,24 +19,24 @@
 	// HEALTHS
 	var/health_brute = 1
 	var/health_burn = 1
-	// no medicine for radio birds
```
useful comment

FlameArrow57 replied:
Not sure if its needed when its a parent variable? Does a whole lot more than no healing too

amylizzle replied:
Maybe change the comment to reflect that then. I just don't want future people wondering why we disabled metabolism without explanation

amylizzle commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -19,24 +19,24 @@
-	use_stamina = 0 //haha no
+	use_stamina = FALSE
 
-	can_lie = 0 // no rotate when dead
```
useful comment

FlameArrow57 replied:
Idk, not sure if it's needed when can_lie already does a whole lot of other stuff

amylizzle commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -142,33 +142,30 @@
+		return TRUE
+	return FALSE
 
-// common features all flock bots get include automatic fire extinguishers
```
useful comment

stonepillars replied:
This one is probably ok

FlameArrow57 replied:
I removed it as there's already a comment in Life() for exinquishers

amylizzle commented on code/mob/living/critter/flockcritter_parent.dm:
```diff
@@ -187,12 +184,10 @@
 // all flock bots should have the ability to rally somewhere (it's applicable to anything with flock AI)
 /mob/living/critter/flock/proc/rally(atom/movable/target)
 	if(src.is_npc)
-		// tell the npc AI to go after the target
```
useful comment

amylizzle commented on code/mob/living/intangible/flock/flockmind.dm:
```diff
@@ -130,7 +116,7 @@
 	O.icon = src.icon
 	O.icon_state = "flockmind-ghost"
 	O.pixel_y = initial(O.pixel_y) // WHY DO I NEED TO DO THIS TOO I DON'T EVEN ANIMATE THE PIXEL_Y
-	animate_bumble(O) // bob up and down
```
useful comment

amylizzle commented on code/obj/flock/furniture.dm:
```diff
@@ -343,7 +337,6 @@
 	src.AddComponent(/datum/component/flock_protection)
 
 
-// flockdrones can always move through
```
useful comment

amylizzle commented on code/obj/flock/structure/collector.dm:
```diff
@@ -1,21 +1,21 @@
 	var/list/turf/simulated/floor/feather/connectedto = list()
 
- //needed for passthrough
-	// drones can pass through this, might change this later, as balance point
```
useful comment

FlameArrow57 replied:
I removed it as there's already a doc comment on /obj/flock_structure for this

amylizzle replied:
👍

amylizzle commented on code/obj/flock/structure/flock_structure_ghost.dm:
```diff
@@ -47,6 +46,16 @@
 		flock_speak(null, "ERROR: Build area is blocked by an obstruction.", flock)
 		return
 
+/obj/flock_structure/ghost/proc/flock_is_blocked_turf(var/turf/T)
```
I think this might be from a bad merge, I made this a global function.

amylizzle commented on code/obj/flock/structure/flock_structure_parent.dm:
```diff
@@ -126,25 +125,22 @@
 /obj/flock_structure/proc/deconstruct()
-	//you can have half your resources back
 	visible_message("<span class='alert'>[src.name] suddenly dissolves!</span>")
-	var/refund = round((src.health/src.health_max) * 0.5 * src.resourcecost) //this is floor(), depsite the name
```
useful comment

amylizzle commented on code/obj/flock/structure/flock_structure_parent.dm:
```diff
@@ -126,25 +125,22 @@
 
 
 /obj/flock_structure/proc/gib(atom/location)
-	// no parent calling, we're going to completely override this
```
useful comment

amylizzle commented on code/obj/flock/structure/flock_structure_parent.dm:
```diff
@@ -234,7 +229,7 @@
 
 	src.report_attack()
 
-	var/damage = round((P.power*P.proj_data.ks_ratio), 1.0) // stuns will do nothing
```
useful comment

amylizzle commented on code/obj/flock/structure/relay.dm:
```diff
@@ -18,11 +16,11 @@
 	pixel_y = -64
 	bound_x = -64
 	bound_y = -64
-	layer = EFFECTS_LAYER_BASE //big spooky thing needs to render over everything
```
useful comment, layers are not super obvious at a glance

amylizzle commented on code/obj/flock/structure/relay.dm:
```diff
@@ -32,12 +30,11 @@
 
 /obj/flock_structure/relay/New()
 	..()
-	// no shuttle for you, either destroy the relay or flee when it unleashes
```
useful comment

amylizzle commented on code/obj/flock/structure/relay.dm:
```diff
@@ -49,7 +46,6 @@
 
 /obj/flock_structure/relay/disposing()
 	..()
-	//crew destroyed it, let them call shuttle
```
useful comment

stonepillars replied:
dubiously useful, intent is declared through `disposing` override. e.g. "This does not exist anymore, allow it"

amylizzle commented on code/obj/flock/structure/relay.dm:
```diff
@@ -71,18 +67,16 @@
 	if(elapsed >= charge_time_length/2)
 		if(icon_state == "structure-relay")
 			icon_state = "structure-relay-glow"
-		// halfway point, start playing radio noises at people too
```
useful comment

FlameArrow57 replied:
I think halfway point is explained by the if() above, think the radio noises probably explained below too?

amylizzle commented on code/obj/flock/structure/relay.dm:
```diff
@@ -126,7 +119,6 @@
 
 ///Brick every headset noisily
 /obj/flock_structure/relay/proc/destroy_radios()
-	//mid-tier jank, but it's a nice easy way to get the radio network
```
useful comment, this is indeed jank

amylizzle commented on code/obj/machinery/door/feather.dm:
```diff
@@ -103,7 +111,6 @@
 
 
 /obj/machinery/door/feather/attack_ai(mob/user as mob)
-	// do nothing, AI and borgs can't interface with the door
```
useful comment

FlameArrow57 replied:
Already explained by attack_ai() maybe?

amylizzle reviewed (changes requested):
Sorry, I maybe over-comment my code, but it seems quite a few people use this to learn how to code so it's good to comment where appropriate. Most of the stuff I'd want left in just explains intent, or clarifies what code does where it isn't super obvious. 

Also I think there's a bad merge resolution in there, so that too.

FlameArrow57 commented on code/datums/flock/flock.dm:
```diff
@@ -1,10 +1,11 @@
 /////////////////////////////
 // FLOCK DATUM
 /////////////////////////////
-// used to manage and share information between members of a flock/nest
```
I kept it, I just reworded it a bit and moved it

amylizzle replied:
👍

FlameArrow57 commented on code/datums/abilities/flock/flockmind.dm:
```diff
@@ -286,23 +282,21 @@
 	var/list/targets = list()
 	for(var/mob/living/M in range(10, holder.owner))
 		if(isflock(M))
-			continue // don't affect us or our flockdrones, yeesh
```
Idk, says isflock() right above?

amylizzle replied:
Intent though, explains what we're achieving by checking for flock

FlameArrow57 replied:
Ok I actually just tested this out, doesn't seem you can actually affect flock because they don't have ear slots, so idk if even needed?

amylizzle replied:
lul

FlameArrow57 commented on code/datums/flock/flock.dm:
```diff
@@ -81,18 +80,15 @@
 			var/mob/living/intangible/flock/trace/T = locate(params["origin"])
 			if(T)
 				if(tgui_alert(user, "This will destroy the Flocktrace. Are you sure you want to do this?", "Confirmation", list("Yes", "No")) == "Yes")
-					// if they're in a drone, kick them out
```
Idk, seems like just a rewording of the release_control() below?

FlameArrow57 commented on code/mob/living/critter/ai/flock/flocktasks.dm:
```diff
@@ -163,21 +170,20 @@ butcher
 	for(var/turf/simulated/floor/T in view(max_dist, holder.owner))
 		if(!isfeathertile(T))
 			if(F?.flock && !F.flock.isTurfFree(T, F.real_name))
-				continue // this tile's been claimed by someone else
```
Think this should be explained by the if()?

FlameArrow57 replied:
I just put a comment on isTurfFree() to make it a little more clear

FlameArrow57 commented on code/datums/flock/flock.dm:
```diff
@@ -56,21 +56,20 @@
 			if(origin)
 				var/turf/T = get_turf(origin)
 				if(T.z != Z_LEVEL_STATION)
-					// make sure they're not trying to spoof data and jump into a z-level they ought not to go
```
Idk, seems already explained by the if() and boutput() message?

amylizzle replied:
Yeah maybe, but it doesn't hurt to do it in plain English also

FlameArrow57 commented on code/datums/abilities/flock/flockmind.dm:
```diff
@@ -286,23 +282,21 @@
 			continue
 		var/obj/item/device/radio/R = M.ears
 		if(istype(R) && R.listening)
-			// your headset's on, you're fair game!!
```
Should be explained by the assignment and if() right above?

amylizzle replied:
Again intent. It's not unclear, but I'm asking myself if a complete coding newbie could follow this better with commenta

FlameArrow57 commented on code/datums/flock/flock.dm:
```diff
@@ -56,21 +56,20 @@
 					user.set_loc(T)
 		if("rally")
 			var/mob/living/critter/flock/C = locate(params["origin"])
-			if(C?.flock == src) // no ordering other flocks' drones around
```
Idk seems already explained by what's checked for, maybe more of a sanity check message would be better?

amylizzle replied:
So a lot of these either clarify intent or explain in plain English what the code is doing. To my mind, that's what comments are for - helping someone entirely unfamiliar with the language/codebase follow what's going on. I'm not going to respond to all of these because lazy, but with that criteria use your best judgement.

FlameArrow57 commented on code/datums/flock/flock.dm:
```diff
@@ -462,15 +455,14 @@
 	if(!M)
 		return
 	if (isvehicle(M))
-		for (var/mob/occupant in M) //yes we are blaming the passenger
```
Clarified intent a bit more

stonepillars commented:
This more or less looks okay ping me if you ever want this merged because comments are subjective and I really don't care

amylizzle reviewed (approved):
it compiles :+1:

stonepillars reviewed (approved):
if this explodes im metagrudging you on classic
